lore/learningobjects/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+
+class Course(models.Model):
+    repository = models.ForeignKey(Repository)
+    org = models.TextField()
+    course_number = models.IntegerField()
+    semester = models.TextField()
+    import_date = models.DateField()
+    imported_by = models.TextField()
+
+class LearningObject(models.Model):
+    course = models.ForeignKey(Course)
+    learning_object_type = models.ForeignKey(LearningObjectType)
```

Comment:
We should change LearningObjectType to a string since it's defined below this point